package.json

```diff
@@ -1,57 +1,60 @@
 {
-    "name": "nodecg-io-cli",
-    "version": "0.1.0",
-    "description": "The CLI to install and manage nodecg-io installations. Also helps you with nodecg-io related development.",
-    "main": "build/index.js",
-    "scripts": {
-        "build": "tsc -b",
-        "run": "tsc -b && node build/index.js",
-        "watch": "tsc -b -w",
-        "lint": "eslint src --ext ts",
-        "format": "prettier --write \"./**/*.{ts,json,md}\"",
-        "format-pre-commit": "pretty-quick --staged --pattern '*/**/*.{ts,json,md}'",
-        "prepare": "husky install"
-    },
-    "bin": {
-        "nodecg-io": "index.js"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/codeoverflow-org/nodecg-io-cli.git"
-    },
-    "keywords": [
-        "nodecg-io"
-    ],
-    "author": "CodeOverflow Team",
-    "license": "MIT",
-    "bugs": {
-        "url": "https://github.com/codeoverflow-org/nodecg-io-cli/issues"
-    },
-    "homepage": "https://github.com/codeoverflow-org/nodecg-io-cli#readme",
-    "devDependencies": {
-        "@types/gunzip-maybe": "^1.4.0",
-        "@types/inquirer": "^7.3.1",
-        "@types/node": "^14.14.31",
-        "@types/semver": "^7.3.4",
-        "@types/tar-fs": "^2.0.0",
-        "@types/yargs": "^16.0.0",
-        "@typescript-eslint/eslint-plugin": "^4.17.0",
-        "@typescript-eslint/parser": "^4.17.0",
-        "eslint": "^7.21.0",
-        "husky": "^5.1.3",
-        "prettier": "^2.2.1",
-        "pretty-quick": "^3.1.0",
-        "typescript": "^4.2.2"
-    },
-    "dependencies": {
-        "axios": "^0.21.1",
-        "chalk": "^4.1.0",
-        "find-up": "^5.0.0",
-        "gunzip-maybe": "^1.4.2",
-        "inquirer": "^8.0.0",
-        "isomorphic-git": "^1.8.1",
-        "semver": "^5.3.0",
-        "tar-fs": "^1.16.3",
-        "yargs": "^16.2.0"
-    }
+  "name": "nodecg-io-cli",
```

Comment:
Would be nice if you could switch the identation back to four spaces, as it is configured in our `.prettierrc.js` and done everywhere else in the repo :wink: